README.md

```diff
@@ -710,6 +710,7 @@ Usage of kube2iam:
       --metrics-port string                   Metrics server http port (default: same as kube2iam server port) (default "8181")
       --namespace-key string                  Namespace annotation key used to retrieve the IAM roles allowed (value in annotation should be json array) (default "iam.amazonaws.com/allowed-roles")
       --cache-resync-period                   Refresh interval for pod and namespace caches
+      --deal-with-duplicated-ip-in-cache       Queries the k8s api server when the pod's cache contains more than a pod with the same IP
```

Comment:
```suggestion
      --resolve-duplicate-cache-ips       Queries the k8s api server to find the source of truth when the pod cache contains multiple pods with the same IP
```

I agree with previous discussions that the argument name is not clear enough. Let me know what you think about the suggestion. Ideally both the argument name and the argument help string should inform you of the purpose of the argument to some degree, and I believe that `--deal-with-duplicated-ip-in-cache` does not sufficiently do that because the verb "deal" is not as expressive as "resolve".